src/components/Card/StyledCard.js

```diff
@@ -0,0 +1,233 @@
+import React, { PropTypes } from 'react'
+import { keys } from 'lodash'
+import styled, { css } from 'styled-components'
+import { CourseMeta, CourseHeader } from './CourseCard'
+import { LessonMeta, LessonHeader } from './LessonCard'
+import { PlaylistMeta, PlaylistHeader } from './PlaylistCard'
+import Playlist from '../Playlist/'
+import { buildPlaylistMeta } from '../../utils/Playlist'
+import { secondsToString } from '../../utils/Time'
+
+const commonCardClasses = 'relative'
+const commonInnerClasses = 'flex flex-column items-center bg-white navy relative z-1 br2'
+const enhancedInnerClasses = `${commonInnerClasses} overflow-hidden pa4 pointer`
+const footerClasses = 'flex justify-between self-stretch items-center' 
+const pillClasses = 'f6 lh-title ttu fw6 ph3 pv1 br-pill tracked'
+const orangePillClasses = `${pillClasses} orange bg-tag-orange`
+const bluePillClasses = `${pillClasses} blue bg-tag-blue`
+const greenPillClasses = `${pillClasses} dark-green bg-tag-turquoise tracked`
+const titleHeadingClasses = 'f3 tc mt4 mb2 avenir fw5'
+const instructorNameClasses = 'f6 dark-gray o-50 mb4 tc'
+
+const cardTypes = {
+  'course': {
+    'cardStyles': [
+      'max-width: 380px;',
+      'box-sizing: border-box;'
+    ],
+    'cardClasses' : 'relative',
```

Comment:
`cardClasses` are from `tachyons-egghead` and will be used in the component's `className={...}`